spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/web/WebConfiguration.java

```diff
@@ -128,7 +130,7 @@ public Jackson2ObjectMapperBuilderCustomizer dataflowObjectMapperBuilderCustomiz
 			// https://github.com/spring-projects/spring-hateoas/issues/333
 			builder.mixIn(StepExecution.class, StepExecutionJacksonMixIn.class);
 			builder.mixIn(ExecutionContext.class, ExecutionContextJacksonMixIn.class);
-			builder.modules(new JavaTimeModule());
+			builder.modules(new JavaTimeModule(), new Jdk8Module());
```

Comment:
what did this do?

Reply:
That parses `Optional`, etc , no longer needed for this, but doesn't hurt